modules/nf-core/fgbio/callmolecularconsensusreads/main.nf

```diff
@@ -19,19 +21,42 @@ process FGBIO_CALLMOLECULARCONSENSUSREADS {
 
     script:
     def args = task.ext.args ?: ''
-    def prefix = task.ext.prefix ?: "${meta.id}"
+    def prefix = task.ext.prefix ?: "${meta.id}_consensus_unmapped"
```

Comment:
Why did you add this custom suffix here?

Reply:
It needs some kind of name renaming to prevent it from colliding with the input file. Some of the subtools already done this, some don't. Although this one is now setting a double prefix in the output file, so that needs fixing.

Reply:
You can add a check for that (a lot of modules do this already) => https://nf-co.re/docs/guidelines/components/modules#command-file-output-naming

Reply:
Yes, it needs the checks whatever.
I think it is better to have a default prefix that doesn't require you to add one.

Reply:
I'm not one to argue with that, but I don't think diverting from the guidelines is a good idea, they're here for a reason. You can maybe bring this up on Slack?